src/components/ButtonPanel.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Button from './Button';
+
+export default function ButtonPanel() {
+  const Panelstyles = {
+    display: 'flex',
+    border: '4px solid yellow',
+    flexWrap: 'wrap',
+  };
+
+  const groupStyle = {
+    display: 'flex',
+    width: '100%',
+    height: 100,
+    border: '4px solid green',
+    margin: 2,
+  };
+
+  return (
+    <div style={Panelstyles}>
+      <div style={groupStyle} className="Group-1">
+        <Button color="" name="AC" />
+        <Button color="" name="+/-" />
+        <Button color="" name="%" />
+        <Button color="" name="÷" />
+      </div>
+      <div style={groupStyle} className="Group-2">
+        <Button color="" name="7" />
+        <Button color="" name="8" />
+        <Button color="" name="9" />
```

Comment:
- You can pass the grey color here to the prop `color`

Reply:
fixed